.github/workflows/tests.yml

```diff
@@ -9,13 +9,17 @@ jobs:
   test:
     runs-on: ubuntu-20.04
 
+    strategy:
+      matrix:
+        node-version: [14, 16, 18]
```

Comment:
What about 20?

Reply:
Didn't know that version exists lol. Will add it in another PR. Thanks!